### DISCUSSION TITLE
Switch between theme colour

### DISCUSSION THREAD

**Discussion by lstables** [rappasoft/laravel-livewire-tables#707] [Q&A]
Is there a way sing the Tailwind theme to switch between dark mode and light mode?

**Comment by insulae** [comment]
you can trying this:
https://rappasoft.com/blog/enabling-dark-mode-on-laravel-boilerplate-pro

in my case I use dark mode in my computer but I want to make the app in light mode, to do that only need to put in the tailwind.conf.js :

` darkMode: ''`
